cnab/app/run

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+action=$CNAB_ACTION
+name=$CNAB_INSTALLATION_NAME 
+echo $NODE_PORT
+use_node_port=""
+if [ "$NODE_PORT" = true ]; then
+    use_node_port="--set gateways.istio-ingressgateway.type=NodePort"
+fi
+
+case $action in
+    install)
+        echo "Installing helm client"
+        helm init --client-only
+
+        echo "Installing Istio"
+        helm install -n istio /charts/istio --namespace istio-system ${use_node_port} --wait
+
+        echo "Installing riff"
+        helm install -n riff /charts/riff
+        ;;
+    uninstall)
+        echo "uninstall action"
+        helm delete --purge riff
+        helm delete --purge istio
+        ;;
+    upgrade)
+        echo "Upgrade action"
+        helm upgrade $name $mychart
```

Comment:
`$name` is not parameterized on install, it'd strange that we start to parameterize it here. Also $mychart appears to be unbound